src/runtime/virtcontainers/pkg/annotations/annotations.go

```diff
@@ -245,7 +245,7 @@ const (
 	EnableRootlessHypervisor = kataAnnotHypervisorPrefix + "rootless"
 
 	// Initdata is the initdata passed in when CreateVM
-	Initdata = kataConfAnnotationsPrefix + "runtime.cc_init_data"
+	Initdata = kataConfAnnotationsPrefix + "hypervisor.cc_init_data"
```

Comment:
So when this merges we'll also need to update the CAA code and anywhere else that uses the init_data annotation and probably remember to add a breaking change in the release notes of kata 3.21 and CoCo 0.16.0 to document this

Reply:
I've updated the commit message. Thanks!

Reply:
Cool !